.gitmessage.txt

```diff
@@ -14,4 +14,4 @@ component: summarize changes in 50 char or less
 # Multiple `Co-Authored-by` lines are allowed.
 Co-Authored-by: [Your Full Name] <[your-email]>
 
-# https://github.com/inveniosoftware/invenio/blob/master/CONTRIBUTING.rst#commit-messages
\ No newline at end of file
+# https://github.com/inveniosoftware/invenio/blob/master/CONTRIBUTING.rst#commit-messages
```

Comment:
You have to fetch changes before your commit and resolve conflicts. Merge is not a good idea.